server/node.go

```diff
@@ -292,6 +298,12 @@ func (n *Node) initStores(engines []engine.Engine, stopper *util.Stopper) error
 		n.initNodeID(0)
 	}
 
+	// Now that we know our ClusterID and NodeID, store it in our LocalSender,
+	// which embeds a context.Context as a base context for requests.
+	log.Add(n.lSender,
```

Comment:
@spencerkimball it seemed better than having a second thing called ctx in the node. But agreed, better to refer to its full name.